### PR TITLE
Sk0 34

### PR DESCRIPTION
**Pull request by Jankita58**

### CODE REVIEW

**Review by markkovari — APPROVED** [review]

File: sk0-front/src/app/app.component.ts [hunk 1]
```diff
@@ -13,9 +13,16 @@ import { CommonModule } from "@angular/common";
...
+    private authService: AuthService,
+    private navCtrl: NavController
+  ) {}
 
   logout() {
     this.authService.logout();
   }
+
+  navigateToUserEdit() {
+    this.navCtrl.navigateForward("/edit");
```
Comment (rolcho) [inline sk0-front/src/app/app.component.ts]: our endpoint will be "/profile"

File: sk0-front/src/app/app.component.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.capitalize-text {
```
Comment (rolcho) [inline sk0-front/src/app/app.component.scss]: do we need shouting text?

Comment (Jankita58) [reply sk0-front/src/app/app.component.scss]: Only the fist letter was uppercase, but  now found a better way to get around ionic button's default uppercase lettering.

File: sk0-front/src/app/app.component.ts (hunk 1 above)
Comment (Jankita58) [reply sk0-front/src/app/app.component.ts]: On it, thanks!

Comment (markkovari) [reply sk0-front/src/app/app.component.ts]: Well technically it is a path, but yes.

Comment (rolcho) [reply sk0-front/src/app/app.component.ts]: Sry

File: sk0-front/src/app/app.component.scss (hunk 2 above)
Comment (rolcho) [reply sk0-front/src/app/app.component.scss]: Let it style the ionic way is my opinion

**Review by rolcho — APPROVED** [review]

**Review by bendesbeus — APPROVED** [review]

**Pull request merged**